include/ajax.tickets.php

```diff
@@ -394,6 +394,11 @@ function changeTicketStatus($tid, $status, $id=0) {
                 if (!$role->hasPerm(TicketModel::PERM_CLOSE))
                     Http::response(403, 'Access denied');
                 $state = 'closed';
+
+                // Check if ticket is closeable
+                if (($warn=$ticket->isCloseable()) && $warn !== true)
```

Comment:
`if (is_string($warn = $ticket->isCloseable()))` would be *much* more readable. At first glance, I read `if (true && false)`